src/cloud/social/provider.ts

```diff
@@ -58,6 +58,14 @@ interface SavedContact {
   description?: string;
 }
 
+var UserStatus = {
```

Comment:
Yes! I have parallel changes in progress in a uproxy branch: https://github.com/uProxy/uproxy/tree/lucyhe-cloudinvite